qos-service-impl/src/main/java/com/palantir/atlasdb/qos/config/QosClientLimitsConfig.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.qos.config;
+
+import java.io.Serializable;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableQosClientLimitsConfig.class)
+@JsonSerialize(as = ImmutableQosClientLimitsConfig.class)
+public abstract class QosClientLimitsConfig implements Serializable {
+    // These numbers come from the metrics released and are scaled by a factor of 2,
+    // we should probably scale up more or set to no limit?
+    public static final long BYTES_READ_PER_SECOND_PER_CLIENT = 500_000L;
+    public static final long BYTES_WRITTEN_PER_SECOND_PER_CLIENT = 1_000_000L;
```

Comment:
is this right, this seems quite low -- 500 kilobytes per second read and 1000 kilobytes per second write?

Reply:
True, just looked at the metrics again (they were aggregated incorrectly earlier), changing the defaults to 50MB and 10MB for reads and writes, respectively.